index.js

```diff
@@ -1,10 +1,23 @@
 var createDecoder = require('./decoder');
+var parseLevel7 = require('./dbase-level-7')
+
 function dbfHeader(data) {
   var out = {};
+
+  switch(data[0] & 0x07) {
+    case 0x03:
```

Comment:
lets not get all fancy here, `0x03` is also know as...3, same with `0x07` and `0x04`